client/hid_client.c

```diff
@@ -36,10 +34,14 @@ struct {
 	usb_desc_list_t strprod;
 
 	usb_desc_list_t hidreport;
-} hid_common;
+} hid_common = { 0 };
```

Comment:
If variable is located in .bss section it should be zeroed by kernel. I think that in this case it is redundant.

Reply:
Ack.

Reply:
Done.